pom.xml

```diff
@@ -91,6 +91,12 @@
             <version>${mule.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mule.runtime</groupId>
+            <artifactId>mule-api</artifactId>
+            <version>1.2.4-SNAPSHOT</version>
```

Comment:
use a property like ${muleApiVersion} or similar name in place of the hardcoded version  to be able to update it using the jenkins jobs